rm/rm-policy/rm-policy-scheduler/src/main/java/org/ow2/proactive/scheduler/resourcemanager/nodesource/policy/SchedulerAwarePolicy.java

```diff
@@ -117,6 +117,9 @@ public void shutdown(Client initiator) {
     public void jobStateUpdatedEvent(NotificationData<JobInfo> notification) {
     }
 
+    public void jobUpdatedFullDataEvent(JobState job) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1161)